packages/opds-web-client/src/__tests__/OPDSDataAdaptor-test.ts

```diff
@@ -102,32 +102,32 @@ describe("OPDSDataAdapter", () => {
     let book = collection.lanes[0].books[0];
     expect(book.id).to.equal(entry.id);
     expect(book.title).to.equal(entry.title);
-    expect(book.authors.length).to.equal(2);
-    expect(book.authors[0]).to.equal(entry.authors[0].name);
-    expect(book.authors[1]).to.equal(entry.authors[1].name);
-    expect(book.contributors.length).to.equal(1);
-    expect(book.contributors[0]).to.equal(entry.contributors[0].name);
-    expect(book.series.name).to.equal(entry.series.name);
-    expect(book.series.position).to.equal(entry.series.position);
+    expect(book.authors?.length).to.equal(2);
+    expect(book.authors?.[0]).to.equal(entry.authors[0].name);
```

Comment:
You can see that with `strictNullChecks` enabled, typescript is now catching all of these previously possibly undefined property access errors, which is why I have to use optional chaining to make sure I'm not going to get an error at runtime. 

Not super helpful in these tests, but those kind of errors would crash the app in production